### PR TITLE
[zh-cn] Resync some MySQL example YAMLs

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/zh-cn/examples/application/mysql/mysql-configmap.yaml [hunk 1]
```diff
@@ -4,15 +4,14 @@ metadata:
   name: mysql
   labels:
     app: mysql
+    app.kubernetes.io/name: mysql
 data:
   primary.cnf: |
-    # Apply this config only on the primary.
+    # 仅在主服务器上应用此配置
```
Comment (windsonsea) [inline content/zh-cn/examples/application/mysql/mysql-configmap.yaml:10]: Could I ask why the period `。` is removed from Chinese? Thanks.

Comment (tengqm) [reply content/zh-cn/examples/application/mysql/mysql-configmap.yaml:10]: It doesn't have to be removed. Keep it there if you wish.
My feeling is that a period in this situation looks weird.

**Comment by windsonsea** [comment]
Enjoy your text.

/lgtm

**Review by howieyuen — APPROVED** [review]
/approve

**Pull request merged**